Postman Test Suite/Record Ownership Test/Record Ownership Manual Setup Scripts.sql

```diff
@@ -0,0 +1,82 @@
+
+
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901044')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+SELECT @OwnershipToken_OwnershipTokenId = OwnershipTokenId FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_Two')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+UPDATE [dbo].[ApiClients]
+   SET CreatorOwnershipTokenId_OwnershipTokenId = @OwnershipToken_OwnershipTokenId
+ WHERE ApiClientId =@ApiClient_ApiClientId
+
+
+SELECT CreatorOwnershipTokenId_OwnershipTokenId,*  FROM [dbo].[ApiClients]  where Name in ('255901044','255901001')
+SELECT *  FROM [dbo].[ApiClientOwnershipTokens] where ApiClient_ApiClientId in ( SELECT ApiClientId  FROM [dbo].[ApiClients]  where Name in ('255901044','255901001'))
+SELECT *  FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_One','OwnershipToken_Two')
+
+USE [EdFi_Security_Test]
+Go
+
+DECLARE @AuthorizationStrategyId INT;
+
+INSERT INTO [dbo].[AuthorizationStrategies]   ([DisplayName],[AuthorizationStrategyName],[Application_ApplicationId]) VALUES ('Ownership Based','OwnershipBased', 1)
+
+SELECT @AuthorizationStrategyId=AuthorizationStrategyId from [dbo].[AuthorizationStrategies] where [AuthorizationStrategyName]='OwnershipBased'
+
+
+ --education organization resource claims for read update and delete
+ INSERT INTO  [dbo].[ResourceClaimActionAuthorizationStrategies]
+ ([ResourceClaimActionId],[AuthorizationStrategyId]) 
+ SELECT ResourceClaimActionId,@AuthorizationStrategyId  FROM [dbo].ResourceClaimActions rca 
+INNER JOIN  [dbo].ResourceClaims rc  on rca.ResourceClaimId = rc.ResourceClaimId
+AND ClaimName='http://ed-fi.org/ods/identity/claims/domains/educationOrganizations'
+INNER JOIN  [dbo].Actions a  on a.ActionId= rca.ActionId 
+AND a.ActionName in ('Read','Update','Delete')
+
+ --studentSectionAssociation resource claims for read update and delete
+ INSERT INTO [dbo].[ResourceClaimActions]
+ ([ResourceClaimId],[ActionId])
+ SELECT ResourceClaimId,ActionId  FROM [dbo].ResourceClaims ,  [dbo].Actions 
+WHERE ActionName in ('Read','Update','Delete')
+ AND ClaimName='http://ed-fi.org/ods/identity/claims/studentSectionAssociation'
+ 
+
+  INSERT INTO  [dbo].[ResourceClaimActionAuthorizationStrategies]
+ ([ResourceClaimActionId],[AuthorizationStrategyId]) 
+ SELECT ResourceClaimActionId,@AuthorizationStrategyId  FROM [dbo].ResourceClaimActions rca 
+INNER JOIN  [dbo].ResourceClaims rc  on rca.ResourceClaimId = rc.ResourceClaimId
+AND ClaimName='http://ed-fi.org/ods/identity/claims/studentSectionAssociation'
+INNER JOIN  [dbo].Actions a  on a.ActionId= rca.ActionId 
+AND a.ActionName in ('Read','Update','Delete')
+
+SELECT *  FROM [dbo].ResourceClaimActionAuthorizationStrategies 
```

Comment:
This looks to be diagnostic in nature and shouldn't be in this script.

Reply:
removed it